portfolio/src/main/webapp/main_page_clickables/game.html

```diff
@@ -8,7 +8,7 @@
 </b></p>
 
 <div id="country-score">Score: 0/0</div>
-<button id="random-country-btn" onclick="displayCountry()" type="button" class="btn btn-info btn-sm">Random Country</button>
+<button id="random-country-btn" onclick="getComments()" type="button" class="btn btn-info btn-sm">Random Country</button>
```

Comment:
This is going to need some cleanup ;-)
Looks like testing code -- the `Random Country` button shouldn't be loading comments